### ISSUE TITLE
[Bug]: The column `main.Setting.proxyPassword` does not exist in the current database.

### ISSUE THREAD

**Issue by kneely** [coollabsio/coolify#782]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

`
> api@ start /app
> NODE_ENV=production pnpm prisma migrate deploy && pnpm prisma generate && pnpm prisma db seed && node index.js

Prisma schema loaded from prisma/schema.prisma
Datasource "db": SQLite database "prod.db" at "file:../db/prod.db"

74 migrations found in prisma/migrations

No pending migrations to apply.
Prisma schema loaded from prisma/schema.prisma

✔ Generated Prisma Client (4.6.1 | library) to ./node_modules/.pnpm/@prisma+client@4.6.1_prisma@4.6.1/node_modules/@prisma/client in 3.30s
You can now start using Prisma Client in your code. Reference: https://pris.ly/d/client
```
import { PrismaClient } from '@prisma/client'
const prisma = new PrismaClient()
```
Running seed command `node prisma/seed.js` ...
PrismaClientKnownRequestError:
Invalid `prisma.setting.findFirst()` invocation:

The column `main.Setting.proxyPassword` does not exist in the current database.
    at RequestHandler.handleRequestError (/app/node_modules/.pnpm/@prisma+client@4.6.1_prisma@4.6.1/node_modules/@prisma/client/runtime/index.js:34310:13)
    at RequestHandler.request (/app/node_modules/.pnpm/@prisma+client@4.6.1_prisma@4.6.1/node_modules/@prisma/client/runtime/index.js:34293:12)
    at async PrismaClient._request (/app/node_modules/.pnpm/@prisma+client@4.6.1_prisma@4.6.1/node_modules/@prisma/client/runtime/index.js:35273:16)
    at async main (/app/prisma/seed.js:20:24) {
  code: 'P2022',
  clientVersion: '4.6.1',
  meta: { column: 'main.Setting.proxyPassword' }
}

An error occurred while running the seed command:
Error: Command failed with exit code 1: node prisma/seed.js
 ELIFECYCLE  Command failed with exit code 1.`

### Steps To Reproduce

1. Upgrade to the version 3.12.0.

### Version

3.12.0

**Comment by kneely** [comment]
I don't remember what version I was on previously. I have searched the existing issues and code. I may have found where the issue is in the code.

It looks like it might be in the latest schema migration where "proxyPassword" is dropped.
[migration](https://github.com/coollabsio/coolify/blob/adf82c04ad4f92eff0522b864430c8b99c4893c8/apps/api/prisma/migrations/20221123122143_remote_haproxy_from_db/migration.sql)

**Comment by andrasbacsai** [comment]
Hey, I think you upgraded to the latest version before I officially released it, and it had a weird bug in one of the SQL migration files. The latest and released version (3.12.1) has a fix. You can update it manually by executing the installation script, for example: 

‘wget -q https://get.coollabs.io/coolify/install.sh \
-O install.sh; sudo bash ./install.sh -f’

**Comment by kneely** [comment]
This issue seems to be fixed. I have run into another issue with Prisma. Would you like me to open a new bug or just rename this one to Prisma migrations?

I ran the above command and installed v3.12.1 forcefully. After install I checked the Docker logs and ran into this error. I am still not able to open the web ui.

`✔ Generated Prisma Client (4.6.1 | library) to ./node_modules/.pnpm/@prisma+client@4.6.1_prisma@4.6.1/node_modules/@prisma/client in 846ms
You can now start using Prisma Client in your code. Reference: https://pris.ly/d/client
```
import { PrismaClient } from '@prisma/client'
const prisma = new PrismaClient()
```
Running seed command `node prisma/seed.js` ...
PrismaClientKnownRequestError:
Invalid `prisma.setting.findFirst()` invocation:


Inconsistent column data: Could not convert value "numberOfDockerImagesKeptLocally" of the field `numberOfDockerImagesKeptLocally` to type `Int`.
    at RequestHandler.handleRequestError (/app/node_modules/.pnpm/@prisma+client@4.6.1_prisma@4.6.1/node_modules/@prisma/client/runtime/index.js:34310:13)
    at RequestHandler.request (/app/node_modules/.pnpm/@prisma+client@4.6.1_prisma@4.6.1/node_modules/@prisma/client/runtime/index.js:34293:12)
    at async PrismaClient._request (/app/node_modules/.pnpm/@prisma+client@4.6.1_prisma@4.6.1/node_modules/@prisma/client/runtime/index.js:35273:16)
    at async main (/app/prisma/seed.js:10:24) {
  code: 'P2023',
  clientVersion: '4.6.1',
  meta: {
    message: 'Could not convert value "numberOfDockerImagesKeptLocally" of the field `numberOfDockerImagesKeptLocally` to type `Int`.'
  }
}

An error occurred while running the seed command:
Error: Command failed with exit code 1: node prisma/seed.js
 ELIFECYCLE  Command failed with exit code 1.`
